### ISSUE TITLE
[a11y]: JAWS does not announce ai-chat messages

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#380]
### Browser

Firefox

### Operating System

Windows

### Package version

0.14.1-canary.347

### React version

^1.66.0

### Automated testing tool and ruleset

none

### Assistive technology

JAWS 2025

### Description

JAWS does not announce certain chat messages.
For example, a text message that also holds a link is not announced by JAWS:
```
<clabs-chat-message time-stamp="13:34" slot="message-items">
   <div class="list" slot="message-item-content">
      <div class="list__case1">There are no open cases. To create and manage your cases, go to the <a class="cds--link" href="/test/cases" analytics-name="center case status page link">Manage cases</a> page where you can create a case and track the progress of your reported issue.</div>
   </div>
</clabs-chat-message>
```

In this case, one can move the focus to the link inside the text message, which JAWS will announce, but the text outside the link is never announced.

### Reproduction/example

none

### Steps to reproduce

1. leverage the ai-chat
2. generate chat responses
3. try to have JAWS read out those chat responses

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
